install/bootstrap_livecd_remote_iso.md

```diff
@@ -217,350 +239,350 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
    1. Copy the artifacts into place.
 
       ```bash
-      pit/var/www/ephemeral# mkdir -pv data/{k8s,ceph}
-      pit/var/www/ephemeral# rsync -a -P --delete ./${CSM_RELEASE}/images/kubernetes/ ./data/k8s/
-      pit/var/www/ephemeral# rsync -a -P --delete ./${CSM_RELEASE}/images/storage-ceph/ ./data/ceph/
+      pit:/var/www/ephemeral# mkdir -pv data/{k8s,ceph}
+      pit:/var/www/ephemeral# rsync -a -P --delete ./${CSM_RELEASE}/images/kubernetes/ ./data/k8s/
+      pit:/var/www/ephemeral# rsync -a -P --delete ./${CSM_RELEASE}/images/storage-ceph/ ./data/ceph/
       ```
 
    > The PIT ISO, Helm charts/images, and bootstrap RPMs are now available in the extracted CSM tar.
 
-1. Install/upgrade the CSI and testing RPMs.
+1. Install/upgrade the CSI, check if a newer version was included in the tar-ball.
```

Comment:
Missing phrase here. Also we're not really checking to see if a newer version was included in the tarball as we're using --force.

Reply:
Good catch